bin/build

```diff
@@ -89,16 +90,12 @@ pushd $IMAGE_DIR
   build_images="manageiq-base-worker manageiq-orchestrator manageiq-webserver-worker manageiq-ui-worker"
   for image in $build_images; do
     cmd="docker build --tag $REPO/$image:$TAG \
+                      --format docker \
                       --build-arg FROM_REPO=$REPO \
                       --build-arg FROM_TAG=$TAG \
                       --build-arg RPM_PREFIX=$RPM_PREFIX \
                       $image"
 
-    if [ "$ARCH" = "s390x" ]; then
-      # --format docker is needed to ensure the builds are built with docker v2 manifests
```

Comment:
can we keep this comment somewhere?  If not in code then at least the commit message?